Add iOS 9.3 support and update donation options

palxex opened the pull request:
Adds iOS 9.3 support, should also works for iOS 7/8/9.0-9.1 ( not tested due to lack of device ), not sure for iOS 9.2. Previous design only for iOS9, so pangu8 protection was replaced to pangu 9.2-9.3 protection.

palxex committed:
iOS9.3 patch

ryanb93 commented on CyDelete.xm:
```diff
@@ -325,59 +358,93 @@ static void uninstallClickedForIcon(SBIcon *self) {
+				}
+			}
+			if(contains) {
+				UIAlertController* alert=[UIAlertController alertControllerWithTitle:[NSString stringWithFormat:@"%@ IS uninstalling",[app displayName]] message:@"Please WAIT A SEC" preferredStyle:UIAlertControllerStyleAlert];  
```
Lets make this text a little bit more professional.

`UIAlertController* alert=[UIAlertController alertControllerWithTitle:[NSString stringWithFormat:@"%@ is uninstalling",[app displayName]] message:@"Please wait. This should only take a few seconds." preferredStyle:UIAlertControllerStyleAlert];`

ryanb93 replied:
In addition to this, we should probably be using localized strings here.

ryanb93 commented on CyDelete.xm:
```diff
@@ -397,11 +463,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
 
 	-(NSString *)uninstallAlertTitle {
-		return [NSString stringWithFormat:SBLocalizedString(@"UNINSTALL_ICON_TITLE"),
+		%log;
```
remove this %log

ryanb93 commented on CyDelete.xm:
```diff
@@ -397,11 +463,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
 	}
 
 	-(NSString *)uninstallAlertBody {
+		%log;
```
remove log

ryanb93 commented on CyDelete.xm:
```diff
@@ -427,11 +499,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
 
 	-(NSString *)uninstallAlertConfirmTitle {
-		return SBLocalizedString(@"UNINSTALL_ICON_CONFIRM");
+		%log;
```
remove log

ryanb93 commented on CyDelete.xm:
```diff
@@ -427,11 +499,17 @@ static void uninstallClickedForIcon(SBIcon *self) {
 
 	-(NSString *)uninstallAlertCancelTitle {
-		return SBLocalizedString(@"UNINSTALL_ICON_CANCEL");
+		%log;
```
remove log

ryanb93 commented on README.md:
```diff
@@ -1,4 +1,4 @@
 =========
 
-The original mobile substrate CyDelete tweak created by @DHowett updated for iOS 7 and 8. 
+The original mobile substrate CyDelete tweak created by @DHowett and ported to iOS7/8 by Ryan Burke updated for iOS 9. 
```
A mobile substrate tweak to uninstall applications from the homescreen that have been installed using Cydia. 

### Contributors
CyDelete - [Dustin Howett ](https://github.com/DHowett)
CyDelete 7 & 8 - [Ryan Burke](https://github.com/ryanb93)
CyDelete 9 - [Pal Lockhart](https://github.com/palxex)

ryanb93 commented on CyDelete/Resources/CyDelete.plist:
```diff
@@ -68,7 +68,7 @@
 		</dict>
 		<dict>
```
You're missing entries here to show your donation links for Bitcoin and AliPay :)

palxex commented:
In fact seems this dialog not need to show in latest build. Previous I’m using `apt-get remove then uicache` way, and it disappears too late( almost 10 secs after clicking) so I added the dialog. Then I changed to the new way, first unregister app to hide the icon, then apt-get remove in background; this way the dialog seems unnecessary, since user will have no time on click it again, but I forgot to remove the code. Let me modify it tomorrow :P too late in me time region, almost sun rising, too easy to introduce bug.

> 在 2016年12月16日，上午5:05，Ryan Burke <notifications@github.com> 写道：
> 
> @ryanb93 commented on this pull request.
> 
> In CyDelete.xm <https://github.com/ryanb93/CyDelete/pull/9>:
> 
> >  		SBApplication *app = [icon application];
>  		id pkgName = ownerForSBApplication(app);
>  		if(pkgName != [NSNull null]) {
>  			uninstallClickedForIcon(icon);
>  		}
> -		%orig;
> +		if(cydelete_IOSVersionAbove92() && [[iconPackagesDict allKeys] containsObject:[app bundleIdentifier]]) {
> +			//new mechanism: iOS 9.2 above will not automatically trigger uninstall, we've to call it manually
> +			BOOL contains = NO;
> +			for(CDUninstallDpkgOperation *operation in [uninstallQueue operations]){
> +				if([operation.package isEqualToString:[app bundleIdentifier]]){
> +					contains = YES;
> +					break;
> +				}
> +			}
> +			if(contains) {
> +				UIAlertController* alert=[UIAlertController alertControllerWithTitle:[NSString stringWithFormat:@"%@ IS uninstalling",[app displayName]] message:@"Please WAIT A SEC" preferredStyle:UIAlertControllerStyleAlert];  
> In addition to this, we should probably be using localized strings here.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/ryanb93/CyDelete/pull/9>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AADjbsCB1lJs-nQTT9ht4RkdOtULpFt1ks5rIauvgaJpZM4LOjHZ>.
>

palxex committed:
remove obsolete alert; add donation
